src/main/java/net/dries007/tfc/util/Helpers.java

```diff
@@ -180,6 +180,11 @@ public static <R> Stream<? extends R> flatten(Object t)
         return t instanceof Collection<?> c ? (Stream<? extends R>) c.stream() : (t instanceof Stream<?> s ? (Stream<? extends R>) s : Stream.of((R) t));
     }
 
+    public static String formatPercentage(float percent)
+    {
+        return String.format("%d%%", Math.round(percent));
+    }
+
```

Comment:
This seems like a silly helper method that actually *obscures* what's happening - the `String.format("%d%%"` is important here.